app/src/main/AndroidManifest.xml

```diff
@@ -34,7 +34,7 @@
         android:enableOnBackInvokedCallback="true"
         tools:targetApi="34">
         <activity
-            android:name=".ui.main.web_view.WebViewActivity"
+            android:name=".ui.main.webView.WebViewActivity"
```

Comment:
This should be `webview` based on the convention we're following? But I won't block this PR. You can update this in a later PR if possible.